.travis.yml

```diff
@@ -68,11 +68,12 @@ jobs:
       before_install: skip
       before_script: skip
       script: composer lint
-    - stage: "Shopware 5.2.0"
-      php: "5.6"
-      env:
-        - SHOPWARE_TAG="5.2.0"
-        - SHOPWARE_VERSION="5.2"
+# Deactivated until the fix in SW-528
```

Comment:
Please only deactivate the specific test case in case it is Shopware 5.2.0, and don't disable the entire test suite for it.